src/code/z_parameter.c

```diff
@@ -3749,13 +3751,15 @@ void Interface_Draw(GlobalContext* globalCtx) {
                 gDPPipeSync(OVERLAY_DISP++);
                 gDPSetPrimColor(OVERLAY_DISP++, 0, 0, 255, 255, 255, 255);
                 gDPSetEnvColor(OVERLAY_DISP++, 0, 0, 0, 0);
+                svar5 = gSaveContext.timerX[svar6];
+                svar2 = gSaveContext.timerY[svar6];
                 OVERLAY_DISP =
-                    Gfx_TextureIA8(OVERLAY_DISP, gClockIconTex, 16, 16, ((void)0, gSaveContext.timerX[svar6]),
-                                   ((void)0, gSaveContext.timerY[svar6]) + 2, 16, 16, 1 << 10, 1 << 10);
+                    Gfx_TextureIA8(OVERLAY_DISP, gClockIconTex, 16, 16, svar5,
+                                   svar2 + 2, 16, 16, 1 << 10, 1 << 10);
 
                 // Timer Counter
                 gDPPipeSync(OVERLAY_DISP++);
-                gDPSetCombineMode(OVERLAY_DISP++, G_CC_MODULATEIA_PRIM, G_CC_MODULATEIA_PRIM);
+                gDPSetCombineLERP(OVERLAY_DISP++, 0, 0, 0, PRIMITIVE, TEXEL0, 0, PRIMITIVE, 0, 0, 0, 0, PRIMITIVE, TEXEL0, 0, PRIMITIVE, 0);
```

Comment:
Why was this changed here?

Reply:
This was one of three non-equivalent lines I found. The asm produced by `gDPSetCombineMode` gave the wrong asm, and going back to gfxdis gave me a different macro which gave the correct asm. 

Also I agree there's a little fakery going on. I'm mainly concerned about the `WREG(90)` macro and possibly the same-line meme. Definitely worth searching for a better match in the next round of clean-up

Reply:
Ah ok. Another thing is that you removed ((void)0,X) which is eventually going to be added pretty much everywhere because there is evidence that Nintendo used macros for accessing values in save context but anyways that is something we can find out another time, great job on finally getting this matched

Reply:
its only needed for specific fields of the save context, and i dont think its commonly needed for the ones it was removed from here. so its probably fine
but indeed, we will find out later